wiki/Article_Style_Guide/en.md

```diff
@@ -147,7 +188,7 @@ For a full explanation, see [Beatmap Editor](/wiki/Beatmap_Editor/).
 
 Section headings must **never** have any styles applied to it.
 
-Section headings can have **small icons** in their section headings but never images larger than 1 point (the typical font size height; usually 16 pixels).
+Section headings can have **small icons** in their section headings but never images larger than 1em (the height of a font which is 14 pixels in the osu!wiki).
```

Comment:
alrighty, added new section to explain what to do when an icon is in the heading :s